make/test.mk

```diff
@@ -5,7 +5,7 @@ COV_DIR = $(OUT_DIR)/coverage
 test: generate
 	@echo "running the tests without coverage..."
 	# TODO enable -race flag, there are some DATA RACE need to be fixed
```

Comment:
you can remove this statement now

Reply:
Done https://github.com/codeready-toolchain/registration-service/pull/53/commits/396bbf4b9d28861a6cb93b3c1fd9950105c3617e